internal/executor/artifacts.go

```diff
@@ -121,10 +122,18 @@ func uploadArtifactsAndParseAnnotations(
 		readBuffer := make([]byte, readBufferSize)
 
 		for _, artifactPath := range artifactPaths {
-			if info, err := os.Stat(artifactPath); err == nil && info.IsDir() {
+			info, err := os.Stat(artifactPath)
+
+			if err == nil && info.IsDir() {
 				continue
 			}
 
+			if err == nil && info.Size() > 1 * humanize.MByte {
+				humanFriendlySize := humanize.Bytes(uint64(info.Size()))
+				logUploader.Write([]byte(fmt.Sprintf("Uploading a quite hefty artifact '%s' of size %s",
```

Comment:
Let's do 100Mb as a definition  for hefty. Or just log any sizes more then 10Mb.

Reply:
Or just change the message to not include hefty. 🤔

Reply:
It seemed strange to only log specific artifact uploads and skip others without an explanation (a user might think that some uploads were not done for some reason), so I've added this "hefty" thing.

See https://github.com/cirruslabs/cirrus-ci-agent/pull/62/commits/70a0537cc896d9d98995bc5fe9c2395166dcb257.